kwave/utils/matlab.py

```diff
@@ -75,6 +75,7 @@ def matlab_find(arr: Union[List[int], np.ndarray], val: int = 0, mode: str = "ne
     return np.expand_dims(arr, -1)  # compatibility, n => [n, 1]
 
 
+@typechecker
```

Comment:
This change is not required for this PR but I just added it while I was in the file